site/helm-get-started.md

```diff
@@ -140,7 +140,7 @@ deploying the workloads of `flux-get-started`. After a while you
 will be able to see the Helm releases listed like so:
 
 ```sh
-helm list --namespace demo
+helm list --namespace flux
```

Comment:
This should be `--namespace demo` since we want to show the charts deployed by Flux, not Flux itself.

Reply:
gotcha.    I misunderstood.  I'll add a line to make that more obvious.  After all, "the Helm releases" could be referring to the helm release of flux itself (which is how I read it)